### PR TITLE
th-281: Fixed view types

### PR DESCRIPTION
**Pull request by mV22110702**

### CODE REVIEW

**Review by denred — APPROVED** [review]

**Review by AlexVlaso — APPROVED** [review]

File: backend/src/packages/mail/libs/maps/template-name-to-view.map.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
-import { type IView } from '~/libs/packages/packages.js';
-
 import { TemplateName } from '../enums/enums.js';
-import { type TemplateNameValues } from '../types/types.js';
 import { plainView } from '../views/views.js';
 
-const templateNameToView: Record<TemplateNameValues, IView> = {
+const templateNameToView = {
```
Comment (kalinkaaaa14) [inline backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: add a type for this map

Comment (mV22110702) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts:4]: Why should there be a type for it, if it's inferred from the actual values?

Comment (mV22110702) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts:4]: > add a type for this map
Why should there be a type for it, if it's inferred from the actual values?

Comment (mV22110702) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: > add a type for this map
Why should there be a type if it's inferred and works by the same principle as enum type?

**Review by PavloSukhinin — DISMISSED** [review]

File: backend/src/packages/mail/libs/maps/template-name-to-view.map.ts (hunk 1 above)
Comment (kalinkaaaa14) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: we need to add type here, so we can't assign any other value as one of values from TemplateNameValues to key, and no other to it's value as IView
now I can write whatever I want as a key, and ts won't tell me that something was wrong

Comment (mV22110702) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: Added type, but I had to explicitly specify its keys and values to keep render parameter type.

Comment (kalinkaaaa14) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: @mV22110702 Record<TemplateNameValues, IView> won't it be suitable?

Comment (mV22110702) [reply backend/src/packages/mail/libs/maps/template-name-to-view.map.ts]: IView is generic, so, unfortunately, it's not going to work.

**Pull request merged**